packages/wonder-blocks-toolbar/docs.md

```diff
@@ -0,0 +1,192 @@
+> TODO(scottgrant): Finish building, and write more thorough docs.
+
+## Content toolbar
+
+### Standard view
+
+```js
+<Toolbar
+	contentKind="Video"
```

Comment:
This isn't actually a prop on `Toolbar`.

Reply:
Good eye! This was from an earlier prototype and it's removed now.